src/panoramic/cli/cli.py

```diff
@@ -211,3 +220,188 @@ def scaffold(target_dataset: str, yes: bool):
     )
 
     scaffold_missing_fields_command(target_dataset=target_dataset, yes=yes)
+
+
+@cli.group()
+def connection():
+    """Connection subcommand for managing connections.
+
+    \b
+    All connections are stored in ~/.pano/config file.
+    You can edit this file either manually or using provided commands.
+
+    Expected YAML structure:
+
+    \b
+    name:
+      type: postgres
+      user: my_user
+      password: <password>
+      host: localhost
+      port: 5432
+      database: my_db
+
+    Which effectively results in a connection string similar to: "postgres://my_user@<password>@localhost:5432/my_db".
+
+    \b
+    name: The name of the connection that will be used as reference to specify which connection to use.
+    type: The type of data warehouse you are connecting to.
+    user: The username used for database connection.
+    password: The password used for database connection.
+    host: The hostname (server) used for database connection.
+    port: The port used for database connection.
+    database: The database used for database connection.
+    schema: The schema to build models into by default. Can be overridden by custom models.
+    warehouse: The warehouse to use when building models. (Used by Snowflake)
+    account: The account used for database connection. (Used by Snowflake)
```

Comment:
If we ask for `host`, we can figure this out. Can we remove the explicit ask for this since we've had some confusion around what exactly the "account" value should be (since sometimes it needs to include region as well).

Reply:
We added this field to stay compatible with DBT profiles. Right now you can copy paste existing profile from DBT and it will work. I can change this to map host to account internally. For now I updated the docs to client know what input is expected.

Anyway, let me know what you prefer, compatibility or internal mapping.

Reply:
Btw I am going to merge this and we can change this easily anytime. Just let me know.